library/opensuse

```diff
@@ -1,8 +1,8 @@
 # maintainer: Flavio Castelli <fcastelli@suse.com> (@flavio)
 
 # openSUSE 13.2
-13.2: git://github.com/openSUSE/docker-containers-build@3bbe87081f5d8f89e56fdca093ea24c3ab8b2992 docker
-harlequin: git://github.com/openSUSE/docker-containers-build@3bbe87081f5d8f89e56fdca093ea24c3ab8b2992 docker
+13.2: git://github.com/openSUSE/docker-containers-build746ce782231f9fc58ef21c584854c755bce61a78 @docker
```

Comment:
Looks like the `@` got a little confused here -- should be `git://github.com/openSUSE/docker-containers-build@746ce782231f9fc58ef21c584854c755bce61a78 docker` like the next line, right?